sjb/config/test_cases/test_branch_origin_service_catalog.yml

```diff
@@ -0,0 +1,103 @@
+---
+parent: 'common/test_cases/origin_built_release.yml'
+overrides:
+  junit_analysis: False
+  timer: 'H H * * *'
+  email:
+    - jpeeler@redhat.com
+extensions:
+  actions:
+    - type: "forward_parameters"
+      parameters:
+        - ORIGIN_TARGET_BRANCH
+        - OPENSHIFT_ANSIBLE_TARGET_BRANCH
+    - type: "script"
+      title: "determine the release commit for origin images"
+      repository: "origin"
+      script: |-
+        jobs_repo="/data/src/github.com/openshift/aos-cd-jobs/"
+        git log -1 --pretty=%h > "${jobs_repo}/ORIGIN_COMMIT"
+        source hack/lib/init.sh
+        os::build::rpm::format_nvra > "${jobs_repo}/ORIGIN_BUILT_VERSION"
+    - type: "script"
+      title: "build junitreport"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        OS_BUILD_ENV_PULL_IMAGE=true OS_BUILD_ENV_PRESERVE=_output/local/bin/linux/amd64/junitreport hack/env make build WHAT=tools/junitreport
+    - type: "script"
+      title: "update bindata"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        hack/update-generated-bindata.sh
+    - type: "script"
+      title: "build Origin OC binary with latest RBAC"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        OS_BUILD_ENV_PULL_IMAGE=true OS_BUILD_ENV_PRESERVE=_output/local/bin/linux/amd64/oc hack/env make build WHAT=cmd/oc
```

Comment:
The point of using the `build` stage as the base is to not require a re-build of any Origin bits -- that image is updated after every merge to `master`